lib/interop.js

```diff
@@ -468,7 +468,6 @@ Interop.prototype.toUnifiedPlan = function(desc) {
                         // msid. Note that "recvonly" is a term only defined
                         // for m-lines.
                         uLine.msid = sources[ssrc].msid;
-                        uLine.direction = 'sendrecv';
```

Comment:
PR for tests here: https://github.com/jitsi/sdp-interop/pull/17

I think that recvonly caused problem in FF, because of this you did this hack.
It could be needed yet, but from my point of view this hack should be applied outside of the interop context (typically before using toUnifiedPlan) .